openquake/hazardlib/geo/surface/base.py

```diff
@@ -183,16 +183,23 @@ def get_ry0_distance(self, mesh):
         top_edge = self.mesh[0:1]
         mean_strike = self.get_strike()
 
+        ia = 0
+        ib = -1
+        if (self.__class__.__name__ == 'KiteSurface'):
```

Comment:
Why not the more idiomatic `isinstance(self, KiteSurface)`?

Reply:
I prefer this since I can avoid importing the KiteSurface class into the base one